cli-typescript/src/utils/cmdActions/getWalletInfoAction.ts

```diff
@@ -11,24 +12,24 @@ const getWalletInfoAction = async (symbol: string) => {
     process.exit(1);
   }
 
-  const walletStore = getWalletStore(symbol, false, true);
-  if (!walletStore.exists) {
-    showError({
-      text: `A Wallet file does not exists for ${symbol}.`,
-    });
-    process.exit(1);
-  }
+  try {
+    const walletInfo = await getMETurnkeyServiceClient().getWallet(symbol);
+    if (!walletInfo) {
+      showError({
+        text: `Failed to retrieve wallet information for ${symbol}.`,
+      });
+      process.exit(1);
+    }
 
-  walletStore.read();
-  if (!walletStore.data) {
+    showText('Wallet information retrieved successfully.');
+    console.log(walletInfo);
```

Comment:
Should we use `logger` instead? Nothing secret in `walletInfo`?

Reply:
No nothing secret but I will remove it.